lighthouse-core/config/config.js

```diff
@@ -381,9 +381,13 @@ class Config {
    * @param {!Array<string>=} skipAuditIds
    * @return {!Object<string, {audits: !Array<{id: string}>}>}
    */
-  static filterCategoriesAndAudits(oldCategories, categoryIds = [], auditIds = [],
-      skipAuditIds = []) {
+  static filterCategoriesAndAudits(oldCategories, categoryIds, auditIds, skipAuditIds) {
     const categories = {};
+    const filterByIncludedCategory = !!categoryIds;
+    const filterByIncludedAudit = !!auditIds;
+    categoryIds = categoryIds || [];
+    auditIds = auditIds || [];
```

Comment:
that was what `filterByIncludedCategory` and `filterByIncludedAudit` are attempting to capture which I thought were more clear than `if (categoryIds && ...` 

the first section is all just warnings which `filterByIncludedCategory` have no impact on and adding the conditionals there just seemed like noise